### ISSUE TITLE
Cannot open document from locations outside /home

### ISSUE THREAD

**Issue by danvim** [ONLYOFFICE/snap-desktopeditors#23]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

I can't see any files in the open file dialog if the location is not in `/home`.

These are images taken in OnlyOffice

![image](https://user-images.githubusercontent.com/5255292/66113167-d8026400-e5fe-11e9-8d11-095c999c92e5.png)

![image](https://user-images.githubusercontent.com/5255292/66113284-1009a700-e5ff-11e9-8d05-50bb66b9478f.png)

![image](https://user-images.githubusercontent.com/5255292/66113405-45ae9000-e5ff-11e9-8b4f-8341f3b742a7.png)

This is an image taken in LibreOffice:

![image](https://user-images.githubusercontent.com/5255292/66113220-f5373280-e5fe-11e9-9d34-a381582e1542.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Mount a drive to `/media` and try to open a document inside the location.

**What is the expected behavior?**

Able to open file from any location.

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**

5.4.1.33. I have not tested in versions before. The suite was installed using snap.

**Comment by ShockwaveNN** [comment]
Can confirm on Ubuntu 18.04 and snap `onlyoffice-desktopeditors  5.4.1            38    stable`

**Comment by Rita-Bubnova** [comment]
It is issue 43048 in our private tracker.

**Comment by Igor-Pigalitsyn** [comment]
Hi @danvim ! This bug can be fixed by manually connecting the interface with the command 
```sudo snap connect onlyoffice desktopeditors:removable-media``` 
We are working to ensure that the connection is automatic. Thanks for your feedback's.

**Comment by danvim** [comment]
@Igor-Pigalitsyn I'm sorry, but running this command produces:

```sh
 daniel@daniel-Lenovo-IdeaPad-510-15ISK  ~  sudo snap connect onlyoffice desktopeditors:removable-media
[sudo] password for daniel: 
error: cannot resolve connection, plug snap name is empty
 ✘ daniel@daniel-Lenovo-IdeaPad-510-15ISK  ~  
```

**Comment by Igor-Pigalitsyn** [comment]
@danvim, it's my mistake try
```sudo snap connect onlyoffice-desktopeditors:removable-media```

**Comment by danvim** [comment]
@Igor-Pigalitsyn Thanks! It's working now.

**Comment by fitrandirahayu** [comment]
> @danvim, it's my mistake try
> `sudo snap connect onlyoffice-desktopeditors:removable-media`
i still have an error here. im using elementary os hera

sudo snap connect onlyoffice-desktopeditors:removable-media
error: snap "onlyoffice-desktopeditors" has no plug named "removable-media"

**Comment by KrzysztofW** [comment]
sudo snap connect onlyoffice-desktopeditors:removable-media
is definately not a solution as documents can be stored in random locations which is a perfectly normal situation. The problem here is the strict confiement of the snap package. Clearly this snap should be made with --classic mode.

**Comment by XDmitryK** [comment]
Fixed in version 8.1.1-27

**Comment by Rita-Bubnova** [comment]
I close this issue. Feel free to comment or reopen it if you got further questions.
